VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs

```diff
@@ -364,7 +364,7 @@ public ValueTask<string> RenderTemplateAsync(string templateContent, string temp
                 NewLine = Environment.NewLine,
                 TemplateLoaderLexerOptions = new LexerOptions
                 {
-                    Mode = ScriptMode.Liquid
+                    Mode = ScriptMode.Default
```

Comment:
I'm afraid this mode will broke exists liquid templates if they are still exists. Why did you change it to default?

Reply:
![image](https://user-images.githubusercontent.com/619699/165942326-4bb1640d-0dcd-4ccc-9c43-03a68f02af1b.png)
Seems that Liquid doesn't exists now - it causes build error.